src/docs/development/add-to-app/ios/project-setup.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Integrate a Flutter module into your iOS project
+short-title: Integrate Flutter
+description: Learn how to integrate a Flutter module into your existing iOS project.
+---
+
+TODO(jmagman): add the pre-built framework alternative and review.
+
+## Integration steps
+
+To integrate a Flutter module into your app, use the following steps:
+
+ 1. [Create Flutter Module](#create-a-flutter-module)
+    Creates the Flutter project which is hosted in the existing app.
+ 1. [Add Module Dependency in Existing
+    App](#make-the-host-app-depend-on-the-flutter-module)
+    Sets up XCode to include your Flutter project into the build of your
```

Comment:
I realize this error was already here, but it's "Xcode", not "XCode". (Global change)

Reply:
done. (it seems like it was only spelled this way in this file)